libs/blocks/merch-card/merch-card.js

```diff
@@ -201,6 +201,37 @@ const parseContent = async (el, merchCard) => {
       }
       return;
     }
+    if (tagName === 'H6' && element.firstElementChild?.tagName === 'EM') {
+      const calloutSlot = createTag('div', { slot: 'callout-text' });
+      const calloutContentWrapper = createTag('div');
+      const calloutContent = createTag('div', { class: 'callout-content' });
```

Comment:
A couple of issues here:
1. if the content is text, it should be a paragraph, not div
2. I looked at the syntax and it's a bit confusing: the wrapping div is callout-text, while the actual callout text is callout-content
3. do we really need the extra wrapping div?
<img width="721" alt="Screenshot 2024-07-15 at 13 46 59" src="https://github.com/user-attachments/assets/0ec72e4c-ab0f-4273-83b5-13f69870e1ec">

Reply:
callout can also have link, strikethrough pricing or images. Maybe I can change the name from "callout-text" to "callout-content" to make it more clear. And is it fine if I do that as part of one bug fix which I am taking separately: https://jira.corp.adobe.com/browse/MWPW-154335 . Name change would require mas PR as well.
Also. I am have now removed the class "callout-content". 
Extra div is needed because of the styling requirement. The tooltip image should always come on top right, irrespective of the text being one line or multiline, but we do not want the background grey to span the entire breadth of the text. I had to adjust the display property using extra div to achieve this.